media_driver/linux/common/ddi/media_libva.cpp

```diff
@@ -3915,6 +3915,7 @@ static VAStatus DdiMedia_SyncSurface (
     return DdiMedia_StatusCheck(mediaCtx, surface, render_target);
 }
 
+#if (VA_MINOR_VERSION > 8)
```

Comment:
Can we, please, use libva supplied macro for that:
`#if VA_CHECK_VERSION(1, 8, 0)`

It will be much easier to support and identified such places in a source code.

Reply:
done